runtime/testy/Ansible.py

```diff
@@ -222,16 +228,101 @@ def start(self):
 
         try:
             while True:
-                nextCall = time.time()
                 event = sel.select()
```

Comment:
Also, we went over this together during the fall semester. The way to implement rate-limited UDP receiver class is:

* Block until there is any data (using `select`)
* Read all available data from the kernel, dropping all but the last packet
* Process the single packet

Do you remember discussing this?